disruption_generator/cli.py

```diff
@@ -5,33 +5,31 @@
 import asyncssh
 import sys
 import click
-import logging
+import logging.config
+import yaml
 
 from . import __version__
 from .parsers.experiment_parser import ExperimentParser
 from .listener.alistener import Alistener
 from .trigger.trigger import Trigger
 from os import walk, path
 
+
 logger = logging.getLogger(__name__)
 
 
 @click.command()
 @click.option(
     "--experiments-path",
     "-e",
-    type=click.Path(
-        exists=True, file_okay=False, readable=True, resolve_path=True
-    ),
```

Comment:
This kind of changes have been performed by `black` after I allowed it to have 119 chars on line